src/applications/appeals/10182/utils/submit.js

```diff
@@ -145,8 +142,8 @@ export const createIssueName = ({ attributes } = {}) => {
  * @param {ContestableIssues}
  * @returns {ContestableIssue~Submittable}
  */
-export const getContestableIssues = ({ contestableIssues } = {}) =>
-  (contestableIssues || []).filter(issue => issue[SELECTED]).map(issue => {
+export const getContestableIssues = ({ contestedIssues } = {}) =>
```

Comment:
Would renaming `getContestableIssues` also require a migration?

Reply:
Migrations are only applied to save-in-progress data, then applied to the in-memory form data. This function is called when the submit button is activated, so it's after the save-in-progress data load/migration and immediately before submission.

Reply:
oooooh interesting!